contracts/truefi2/LoanFactory2.sol

```diff
@@ -135,6 +151,11 @@ contract LoanFactory2 is ILoanFactory2, Initializable {
         emit LoanTokenCreated(newToken);
     }
 
+    function createDebtToken() external onlyTCA {
+        address newToken = address(new DebtToken());
+        emit DebtTokenCreated(newToken);
+    }
```

Comment:
Why don't we use minimal proxies for debt tokens?

Reply:
I was gonna do that in another PR, but I can as well here.

Reply:
We decided to do so in another PR.